Controller/Adminhtml/Recipients/Delete.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Pagarme\Pagarme\Controller\Adminhtml\Recipients;
+
+
+use Pagarme\Pagarme\Controller\Adminhtml\Recipients\RecipientAction;
+use Pagarme\Core\Marketplace\Services\RecipientService;
+
+class Delete extends RecipientAction
+{
+    /**
+     * Index action
+     *
+     * @return \Magento\Framework\Controller\ResultInterface
+     * @throws \Exception
+     */
+    public function execute()
+    {
+        $recipientId = (int)$this->getRequest()->getParam('id');
+        if ($recipientId) {
+
+            $recipientService = new RecipientService();
+            $recipientData = $recipientService->findById($recipientId);
+
+            if (!$recipientData || !$recipientData->getId()) {
+                $message = $this->messageFactory->create('error', __('Recipient not exist.'));
```

Comment:
`Recipient does not exist.`
or
`Recipient doesn't exist.`
or
`Recipient not found.`

Reply:
Message changed to `Recipient not found.`